apps/web/composables/useLocalization/__tests__/useLocalization.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { mockNuxtImport } from '@nuxt/test-utils/runtime';
+describe('useLocalization', () => {
+  const { useNuxtApp } = vi.hoisted(() => {
+    return {
+      useNuxtApp: vi.fn().mockImplementation(() => {
+        return {
+          $i18n: {
+            locale: ref('de'),
+            defaultLocale: 'en',
+            strategy: 'prefix_and_default',
+          }
+        }
+      })
+    }
+  });
+
+  mockNuxtImport('useNuxtApp', () => {
+    return useNuxtApp
+  });
+  
+
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('should get url without language prefix default strategy', async () => {
+    const { getCategoryUrlFromRoute } = useLocalization();
+    const res = getCategoryUrlFromRoute('/de/category/subcategory');
+    expect(res).toBe('/category/subcategory');
+  });
+
+  it('should test strategy "prefix_except_default"', () => {
```

Comment:
**question**: Shouldn't this have another mock implementation? As it stands, this test uses the `prefix_and_default` strategy from what I can tell.

Reply:
good catch, done